docs/faq/overview.md

```diff
@@ -0,0 +1,17 @@
+# F.A.Q
+
+## What is the Hedgehog Fabric
+The Hedgehog Fabric is a topology of routers arranged in a spine-leaf architecture. A spine-leaf architecture is a type of Clos topology. In a spine-leaf architectures the leaves are usually placed in racks and connected directly to the servers, and spines are connected only to leaves. In a spine-leaf architecture the fundamental unit of connection is a layer 3 route. The Hedgehog Fabric is managed via Kubernetes objects and custom resource definitions.
+
+## Why is a spine-leaf architecture useful
+A spine-leaf architecture provides more bandwidth for traffic that is passing between servers inside of a data center, other architectures like core-access-aggregation are setup to provide more bandwidth in and out of the data center. A spine-leaf architecture provides multiple paths between nodes which allows for router maintenance and resilience in the case of failures. The spine-leaf architecture allows allows for multiple points of egress via border leaf nodes. In a spine-leaf architecture the unit of connection is a layer 3 route, this is advantageous to manage traffic ingress or egress as well as quality of service for the applications on the network. To manage the availability of routes usually BGP, OSPF, or IS-IS are used.
```

Comment:
```suggestion
When architectures such as core-access-aggregation provide more bandwidth in and out of the data center, a spine-leaf architecture provides more bandwidth for traffic that passes between servers _inside of_ a data center. A spine-leaf architecture comprises multiple paths between nodes, which allows for router maintenance and resilience in the case of failures. A network based on the spine-leaf architecture can contain multiple points of egress via border leaf nodes. In a spine-leaf architecture, the unit of connection is a layer-3 route, which is advantageous to manage traffic ingress or egress as well as quality of service for the applications on the network. The availability of routes is usually ensured by the BGP, OSPF, or IS-IS routing protocols.
```

Why are layer-3 routes advantageous to manage ingress/egress and QoS?

Also: diagrams?

Reply:
I did update this section with a couple of diagrams. I also re-worded the paragraph a bit.